src/ngraph/op/quantized_convolution.cpp

```diff
@@ -0,0 +1,196 @@
+//*****************************************************************************
+
+#include "quantized_convolution.hpp"
+#include <numeric>
+#include "ngraph/coordinate_diff.hpp"
+#include "ngraph/op/convolution.hpp"
+#include "ngraph/util.hpp"
+#include "ngraph/validation_util.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+op::QuantizedConvolution::QuantizedConvolution(const shared_ptr<Node>& input,
+                                               const shared_ptr<Node>& filters,
+                                               const Strides& window_movement_strides,
+                                               const Strides& window_dilation_strides,
+                                               const CoordinateDiff& padding_below,
+                                               const CoordinateDiff& padding_above,
+                                               const Strides& data_dilation_strides,
+                                               const shared_ptr<Node>& input_scale,
+                                               const shared_ptr<Node>& input_zero_point,
+                                               const shared_ptr<Node>& filter_scale,
+                                               const shared_ptr<Node>& filter_zero_point,
+                                               const shared_ptr<Node>& output_scale,
+                                               const shared_ptr<Node>& output_zero_point,
+                                               const element::Type& output_type,
+                                               const AxisSet& input_axes,
+                                               const AxisSet& filter_axes,
+                                               const AxisSet& output_axes)
+    : Op("QuantizedConvolution",
+         check_single_output_args({input,
+                                   filters,
+                                   input_scale,
+                                   input_zero_point,
+                                   filter_scale,
+                                   filter_zero_point,
+                                   output_scale,
+                                   output_zero_point}))
+    , m_window_movement_strides(window_movement_strides)
+    , m_window_dilation_strides(window_dilation_strides)
+    , m_padding_below(padding_below)
+    , m_padding_above(padding_above)
+    , m_data_dilation_strides(data_dilation_strides)
+    , m_output_type(output_type)
+    , m_input_axes(input_axes)
+    , m_filter_axes(filter_axes)
+    , m_output_axes(output_axes)
+{
+    constructor_validate_and_infer_types();
+}
+
+void op::QuantizedConvolution::validate_and_infer_types()
+{
+    enum
+    {
+        INPUT_SCALE = 2,
+        INPUT_ZERO_POINT,
+        FILTER_SCALE,
+        FILTER_ZERO_POINT,
+        OUTPUT_SCALE,
+        OUTPUT_ZERO_POINT
+    };
+
+    NODE_VALIDATION_CHECK(this,
+                          get_input_element_type(INPUT_SCALE).is_real() ||
```

Comment:
`get_input_element_type(INPUT_SCALE).is_dynamic() || get_input_element_type(INPUT_SCALE).is_real() || ...`

Same thing for `FILTER_SCALE` and `OUTPUT_SCALE`.

Reply:
Are the types for `INPUT_SCALE`, `FILTER_SCALE`, and `OUTPUT_SCALE` allowed to be different from each other? If not, we should check them here.